src/main/resources/application.properties

```diff
@@ -0,0 +1,26 @@
+# PROD
+%prod.quarkus.rest-client.onecx_workspace_svc.url=http://onecx-workspace-svc:8080
+
+# DEV
+%dev.quarkus.rest-client.onecx_workspace_svc.url=http://onecx-workspace-svc
+# remove this above
+# BUILD
+quarkus.openapi-generator.codegen.input-base-dir=target/tmp/openapi
+quarkus.openapi-generator.codegen.spec.onecx_workspace_svc_yaml.config-key=onecx_workspace_svc
+quarkus.openapi-generator.codegen.spec.onecx_workspace_svc_yaml.base-package=gen.io.github.onecx.workspace.bff.clients
+quarkus.openapi-generator.codegen.spec.onecx_workspace_svc_yaml.return-response=true
+# INTEGRATION TEST
+%dev.tkit.log.json.enabled=false
```

Comment:
remove this line, defined in the onecx-quarkus-core: https://github.com/onecx/onecx-quarkus/blob/main/extensions/core/runtime/src/main/resources/application.properties